Code/PgSQL/rdkit/CMakeLists.txt

```diff
@@ -0,0 +1,156 @@
+set(EXTENSION "rdkit")
```

Comment:
For what it's worth: this doesn't build on the Mac, at least not using postgresql that is installed with homebrew. I will investigate more later.